packages/web-components/src/dialog-body/dialog-body.ts

```diff
@@ -9,9 +10,20 @@ import { attr, FASTElement } from '@microsoft/fast-element';
  */
 export class DialogBody extends FASTElement {
   /**
-   * @public
-   * Indicates whether the dialog has a title action
+   * Handles click event for the close slot
+   *
+   * @param e - the click event
+   * @internal
    */
-  @attr({ mode: 'boolean', attribute: 'no-title-action' })
-  public noTitleAction: boolean = false;
+  public clickHandler(event: MouseEvent): boolean | void {
```

Comment:
Should this method name be made more descriptive? `clickHandler` sounds a bit too generic for what it actually does.

Reply:
Keeping it as clickHandler to match what we do elsewhere.